pkg/limayaml/limayaml.go

```diff
@@ -69,6 +69,7 @@ const (
 	QEMU VMType = "qemu"
 	VZ   VMType = "vz"
 	WSL2 VMType = "wsl2"
+	EXT  VMType = "ext"
```

Comment:
Needs docs and integration tests.
Probably this should be also marked experimental

Reply:
Probably Incus (or LXD) can be used for emulating an "external" host

Reply:
Experimenting with LXC/LXD, after all we have it in the default install...

Reply:
Removed the integration tests, and the documentation does not have details.